### PR TITLE
integration: add a basic integration test fixture

### PR DESCRIPTION
**Pull request by nicks**
Hello @jazzdan,

Please review the following commits I made in branch nicks/ch455/simpleup:

3b4635165319b81fb6f1e11486985afb27879cec (2018-10-03 15:09:18 -0400)
integration: add a basic integration test fixture
Note that this doesn't run on circleci yet

### CODE REVIEW

File: integration/fixture_test.go [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+	body, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return "", errors.Wrap(err, "Curl")
+	}
+	return string(body), nil
+}
+
+func (f *fixture) CurlUntil(ctx context.Context, url string, expectedContents string) {
+	actualContents := ""
+	for true {
```
Comment (jazzdan) [inline integration/fixture_test.go]: Could this be shortened to just `for {`?

File: integration/fixture_test.go [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+				url, expectedContents, actualContents)
+		}
+
+		actualContents, err := f.Curl(url)
+		if err != nil {
+			continue
+		} else if strings.Contains(actualContents, expectedContents) {
+			return
+		}
+		time.Sleep(200 * time.Millisecond)
```
Comment (jazzdan) [inline integration/fixture_test.go]: The way I'd expect to see this written in Go is a for loop that listens on a couple channels, `time.Tick`, `context.Cancel`, etc. But this is fine too.

File: integration/oneup/main.go [hunk 3]
```diff
@@ -0,0 +1,14 @@
+package main
+
+import (
+	"net/http"
+)
+
+func main() {
+	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
+		msg := "🍄 One-Up! 🍄"
```
Comment (jazzdan) [inline integration/oneup/main.go]: 😆

File: integration/fixture_test.go (hunk 2 above)
Comment (nicks) [reply integration/fixture_test.go]: good points! i cleaned up the loop a bit

File: integration/fixture_test.go (hunk 1 above)
Comment (nicks) [reply integration/fixture_test.go]: done

**Pull request merged**